.github/workflows/deployTest.yml

```diff
@@ -8,28 +8,33 @@ env:
   ARM_CLIENT_SECRET: ${{ secrets.TERRAFORM_ARM_CLIENT_SECRET }}
   ARM_SUBSCRIPTION_ID: ${{ secrets.TERRAFORM_ARM_SUBSCRIPTION_ID }}
   ARM_TENANT_ID: ${{ secrets.TERRAFORM_ARM_TENANT_ID }}
-  ACR_ADMIN_USERNAME: ${{ secrets.ACR_ADMIN_USERNAME }}
-  ACR_ADMIN_PASWORD: ${{ secrets.ACR_ADMIN_PASWORD }}
-  ACR_REPO_URL: ${{ secrets.ACR_REPO_URL }}
   DEPLOY_ENV: test
 
 jobs:
-  deploy-backend:
+  docker-build:
     runs-on: ubuntu-latest
     defaults:
       run:
         working-directory: ./backend
     steps:
       - uses: actions/checkout@v2
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
```

Comment:
This turns out to be unnecessary and therefore presumably useless. Could be a surprise lurking in there, but I am pretty confident we never needed this.